src/components/comments/index.ts

```diff
@@ -1,35 +1,143 @@
 import { Logger } from '../../common/utils';
+import ApiService from '../../services/api';
+import config from '../../services/config';
 import { Comments } from '../../web-components';
 import { BaseComponent } from '../base';
 
+import { Annotation, Comment } from './types';
+
 export class CommentsComponent extends BaseComponent {
   protected name: string;
   protected logger: Logger;
   protected element: Comments;
+  private apiKey: string;
+  private apiUrl: string;
+  private roomId: string;
+  private url: string;
```

Comment:
Is better use this by config object directly.

```suggestion
private url: string;
```